releasenotes-builder/src/main/java/com/github/checkstyle/MainProcess.java

```diff
@@ -100,7 +109,7 @@ public static List<String> validateNotes(Multimap<String, ReleaseNotesMessage> r
                 || releaseNotes.containsKey(Constants.BREAKING_COMPATIBILITY_LABEL);
 
         final List<String> errors = new ArrayList<>();
-        final String errorBeginning = "Validation of release number failed.";
+        final String errorBeginning = "[ERROR] Validation of release number failed.";
```

Comment:
I'm slightly confused by this change and a similar else where. Do we not automatically say each item is an error when we print it and this is an example where we forgot? If so, sounds to me like the error printing should be doing this for us and we shouldn't be doing this manually.

I am ok if we want to delay this.

Reply:
I do not worry about this, I hope we will not update this project after this iteration.
It just need to do a job.